pkg/config/config.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 	PlanPatch          bool   `yaml:"plan_patch"`
 	RepoOwner          string `yaml:"repo_owner"`
 	RepoName           string `yaml:"repo_name"`
+	Output			   string `yaml:"output_file"`
```

Comment:
```suggestion
	Output			   string `yaml:"-"`
```

Hmm. I think `output_file` should be specified by command line argument explicitly.